.kitchen.yml

```diff
@@ -206,6 +206,31 @@ suites:
       - recipe[hadoop::zookeeper]
       - recipe[hadoop::zookeeper_server]
     attributes: { hadoop: { distribution: 'bigtop', distribution_version: '0.8.0' } }
+  - name: bigtop100
+    excludes: debian-6.0.10, ubuntu-14.04
```

Comment:
Let's call that satisfactory. If the DataNode starts, then anything linking to libhdfs will work, and that's the only real native code.